installer.sh

```diff
@@ -24,6 +24,9 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #-
+#
```

Comment:
```suggestion
```
don't think this extra comment line is needed

Reply:
that was a mistake, fixed